src/library_browser.js

```diff
@@ -292,7 +292,8 @@ var LibraryBrowser = {
           ctx = GL.getContext(contextHandle).GLctx;
         }
         // Set the background of the WebGL canvas to black
-        canvas.style.backgroundColor = "black";
+        if (!contextAttributes["alpha"])
+          canvas.style.backgroundColor = "black";
```

Comment:
Yeah, I'm not sure why this is here... looking in git history, it seems that it came from SDL code, and the comment says "SDL windows have a black background by default". And this code got into this file through a refactoring to share code.

So yes, let's remove this here. However, it may break tests. In which case, we can likely fix them by adding the black stuff at the SDL1 level. Might also need it in glut.